### PR TITLE
Remove Darwin specific logic - Closes #317

### LINKED ISSUE #317: Discontinue macOS builds
It is no longer feasible to run a node on macOS.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by abelboldu — DISMISSED** [review]

File: build.sh [hunk 1]
```diff
@@ -59,10 +59,7 @@ cd src || exit 2
 
 echo "Building libreadline7"
 echo "--------------------------------------------------------------------------"
-if [ ! -f "$LIBREADLINE_FILE" ] && [ ! "$(uname -s)" == "Darwin" ]; then
-	exec_cmd "wget $LIBREADLINE_URL -O $LIBREADLINE_FILE"
```
Comment (Nazgolze) [inline build.sh]: still need this guy

Comment (Nazgolze) [reply build.sh]: the wget I mean :)

**Review by Nazgolze — DISMISSED** [review]
missing a wget

**Review by Nazgolze — APPROVED** [review]

**Review by Nazgolze — DISMISSED** [review]
Whoops, I almost missed this.
postgres, we need to force it to link against the packaged libreadline7

(the evil ! "$(uname -s) == "Darwin" lines )

File: build.sh [hunk 2]
```diff
@@ -151,10 +144,6 @@ if [ ! -d "$BUILD_NAME/node_modules" ]; then
 	# Copy jq to binary folder
 	exec_cmd "cp -vf $JQ_DIR/$JQ_OUT $BUILD_NAME/bin/$JQ_OUT"
 
-	if [ ! "$(uname -s)" == "Darwin" ]; then
-	exec_cmd "cp -vf $LIBREADLINE_DIR/out/lib/* $BUILD_NAME/lib"
```
Comment (Nazgolze) [inline build.sh]: This one too. This is all of them

**Review by Nazgolze — DISMISSED** [review]

**Review by Nazgolze — APPROVED** [review]

**Pull request merged**